examples/rcn.py

```diff
@@ -0,0 +1,438 @@
+# ---
+# jupyter:
+#   jupytext:
+#     formats: ipynb,py:percent
+#     text_representation:
+#       extension: .py
+#       format_name: percent
+#       format_version: '1.3'
+#       jupytext_version: 1.13.2
+#   kernelspec:
+#     display_name: Python 3
+#     language: python
+#     name: python3
+# ---
+
+# %% [markdown]
+# # RCN implementation using the PGMax package
+
+# %% [markdown]
+# RCN is a neuroscience based, data-efficient, interpretable object detection/vision system. It is a probabilistic graphical model (PGM) that trains in minutes and generalizes well (for certain traits). It can reason about, and is robust to, occlusion. It can also imagine and interpret scenes.
+#
+# Please refer to the [Science](https://www.cs.jhu.edu/~ayuille/JHUcourses/ProbabilisticModelsOfVisualCognition2020/Lec21/A%20generative%20vision%20model%20that%20trains%20with%20high%20data%20efficiency%20and%20breaks%20text-based%20CAPTCHAs.pdf) paper for more details. In this notebook, we implement a two-level RCN model on a small subset of the MNIST dataset.
```

Comment:
:roll_eyes: 
Can you use this link please?
https://www.science.org/doi/10.1126/science.aag2612

@lazarox do we have open-access link to share?

Reply:
I would also link to the earlier repo https://github.com/vicariousinc/science_rcn since you used that to train the model I assume.

Reply:
Changed the link.

Reply:
And added new link where the model is loaded.